repository-data/site/src/main/resources/hcm-config/hst/configurations/lks/sitemap.yaml

```diff
@@ -25,4 +26,4 @@ definitions:
           /_any_:
             jcr:primaryType: hst:sitemapitem
             hst:componentconfigurationid: hst:pages/minihub
-            hst:relativecontentpath: minihubs/${1}           
```

Comment:
@thojava - Do we still need document driven sitemap for MiniHub `/hst:hee/hst:configurations/lks/hst:sitemap/minihub` and it's associated page configuration `/hst:hee/hst:configurations/lks/hst:pages/minihub` as we moved to channel manager managed MiniHub pages (with `minihub-page` prototype) ? If not, could we get rid of them ?

Reply:
@smanivasag Last time I asked Matt about this but I do not have confirmation about that yet.

Reply:
Ah I see! Thanks for the info. Since, we are supporting channel manager managed MiniHub pages, I think it's safe we can get rid of document driven sitemap that we originally created for MiniHub documents. Otherwise, we will end up with URL aliases i.e. couple of URLs for MiniHub pages which will again require us to add canonical tags to the pages. Hope that makes sense ?

@MattBrooksManifesto @duncanheron - Thoughts ?

Reply:
@smanivasag I will go a head and remove it first and add it back it we need it later.

Reply:
Thanks @thojava - Do we still need `/hst:hee/hst:configurations/lks/hst:pages/minihub` page configuration ? If not, could we get rid of that as well ?